packages/expo-blur/src/BlurView.types.ts

```diff
@@ -17,14 +33,25 @@ export type BlurViewProps = {
   /**
    * A number by which the blur intensity will be divided on Android.
    *
-   * Due to platform differences blurs on Android and iOS vary slightly and might look different
-   * at different intensity levels. This property can be used to fine tune blur intensity on Android to match it
+   * When using experimental blur methods on Android the perceived blur intensity might be different from iOS
```

Comment:
```suggestion
   * When using experimental blur methods on Android, the perceived blur intensity might differ from iOS
```
Small nit to be a bit more concise and missing comma